test/sanity/issue5943-mac-dblite-module-with-node8-crash/test.py

```diff
@@ -0,0 +1,39 @@
+import time
+import os
+import shutil
+import platform
+import sys
+import subprocess
+
+if platform.system() != 'Darwin':
+    print 'Skipped for non Mac platform'
+    sys.exit(0)
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+chrome_options = Options()
+testdir = os.path.dirname(os.path.abspath(__file__))
+chrome_options.add_argument("nwapp=" + testdir)
+node_module = os.path.join(testdir, "node_modules")
+os.chdir(testdir)
+
+try:
+    shutil.rmtree(node_module)
+except:
+    pass
+
+assert(False == os.path.isdir(node_module))
+subprocess.call(["npm", "install", "dblite"])
```

Comment:
installing modules for NW should have some environment variable set as: http://docs.nwjs.io/en/latest/For%20Users/Advanced/Use%20Native%20Node%20Modules/#for-non-lts-releases

Please see how the case 'native-module-buffer' is doing this.

Reply:
Updated the code